appimagebuilder/generator/command_generate.py

```diff
@@ -48,14 +43,20 @@ def __init__(self):
         # configure Recipe Generator
         package_manager_section_generators = []
         if shutil.which("apt-get"):
-            file_package_resolver = FilePackageResolver()
-            package_repository_resolver = PackageRepositoryResolver()
-            apt_section_generator = AptSectionGenerator(
-                file_package_resolver, package_repository_resolver
+            apt_file_package_resolver = apt.FilePackageResolver()
+            apt_package_repository_resolver = apt.PackageRepositoryResolver()
+            apt_section_generator = recipe_sections.AptSectionGenerator(
+                apt_file_package_resolver, apt_package_repository_resolver
             )
             package_manager_section_generators.append(apt_section_generator)
+        if shutil.which("pacman"):
+            pacman_file_package_resolver = pacman.FilePackageResolver()
+            pacman_section_generator = recipe_sections.PacmanSectionGenerator(
+                pacman_file_package_resolver
+            )
+            package_manager_section_generators.append(pacman_section_generator)
 
-        # append files section generator at last as it will catch all the dependencies
+            # append files section generator at last as it will catch all the dependencies
```

Comment:
Hmm, the linter doesn't know what you want. :)

Reply:
It's a very opinionated linter :D